### DISCUSSION TITLE
My local DNS (10.8.0.1) does not work

### DISCUSSION THREAD

**Discussion by wawanUnic** [wg-easy/wg-easy#992] [Q&A]
Hello.
I installed the program on the Raspberry PI.
I haven't used Docker. I set it up according to your instructions (https://github.com/wg-easy/wg-easy/wiki/Using-WireGuard-Easy-without-Docker).
My local DNS (10.8.0.1) does not work.
But DNS 8.8.8.8 works fine for me.
Traffic flows in both directions.
What can you tell me about this?

When I use WG without WG-easy, my local DNS 10.8.0.1 works fine. When setting up the file /etc/wireguard/wg0.conf, I put the lines:
_PostUp   = iptables -A FORWARD -i %i -j ACCEPT; iptables -t nat -A POSTROUTING -o eth0 -j MASQUERADE_
and
_PostDown = iptables -D FORWARD -i %i -j ACCEPT; iptables -t nat -D POSTROUTING -o eth0 -j MASQUERADE_

And in WG-easy you have other lines:
_PostUp =  iptables -t nat -A POSTROUTING -s 10.8.0.0/24 -o eth0 -j MASQUERADE; iptables -A INPUT -p udp -m udp --dport 51820 -j ACCEPT; iptables -A FORWARD -i wg0 -j ACCEPT; iptables -A FORWARD -o wg0 -j ACCEPT;_
and
P_ostDown =  iptables -t nat -D POSTROUTING -s 10.8.0.0/24 -o eth0 -j MASQUERADE; iptables -D INPUT -p udp -m udp --dport 51820 -j ACCEPT; iptables -D FORWARD -i wg0 -j ACCEPT; iptables -D FORWARD -o wg0 -j ACCEPT;_
Maybe this is the problem?
Thank you in advance.

**Comment by pheiduck** [comment]
Well, one iptables rule is for server the other one for the client.
But you shouldn't edit the `wg0.conf` by hand as it gets overwritten by design.
Make sure you edit the service file for your needs where it say **REPLACEME**.
